### PR TITLE
Vocab and Punctuational Changes.

### PR DESCRIPTION
**Pull request by dumbcoder7**
Suggest if you want any change to be reverted.
Also wanted to remove the Contributing section, but hopefully won't come to that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 # Generator
 
-Interface part of Auto-DL. Contains front-end (React) and backend (DJango Rest Framework) servers.
-The backend calls the [DLMML API](https://www.github.com/Auto-DL/DLMML) according to the request it recieved.
+The interface section of Auto-DL. Contains front-end and back-end servers based on React and Django Rest Framework respectively.
```
Comment (ADI10HERO) [inline README.md:17]:
```suggestion
The interface section of Auto-DL contains front-end and back-end servers based on React and Django Rest Framework respectively.
```

Does this look better? @dumbcoder7

**Comment by ADI10HERO** [comment]
Thank you so much for the changes :)

I will be more than happy to merge after your response to the comment.
 

Also adding @shintan777 to have a look to review the changes

**Review by shintan777 — APPROVED** [review]

**Comment by dumbcoder7** [comment]
Yep, committed with your suggested change

**Review by ADI10HERO — APPROVED** [review]
Thank you so much for these updates.

**Pull request merged**